app/apis/hackney_api/repairs_client.rb

```diff
@@ -191,7 +191,7 @@ def self.clear_repairs_cache_for_property(property_reference)
       API_REQUEST_CACHE.delete_matched "*repairs*#{property_reference}*"
     end
 
-    def post_repair_request(name:, phone:, tasks:, priority:, property_ref:, description:, created_by_email:)
+    def post_repair_request(name:, phone:, tasks:, priority:, property_ref:, description:, recharge:, created_by_email:)
```

Comment:
Just for readability, would it make sense to name the parameters with the same names as those that you have in the request hash?

Reply:
They are _kinda_ named after the **RepairRequest** model attributes... I know it's not great, but I'd rather leave it as is, cause renaming those would trigger changes in the rest of the code.

Tbh I think this kind of logic belongs to the models, and not to a separate _API Client_ class. Some reasons for it:

* **RepairsClient** is humongous and contains all API calls (single responsibility principle?)
* the models should be _the way_ to interact with the backend entities because of because validations, transformations and other kind of local logic
* logic that should only be used by a specific model is scattered into two files, making it cumbersome to read the code
* there are places using **RepairsClient** directly to create **Notes**, without going through the model, which

what do you think?